lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1194,6 +1209,8 @@ def embedTTFType42(font, characters, descriptor):
             wObject = self.reserveObject('Type 0 widths')
             toUnicodeMapObject = self.reserveObject('ToUnicode map')
 
+            bbox = [cvt(x, nearest=False) for x in font.bbox]
```

Comment:
Does this need to be calculated so early, and not right before emitting the XObjects?

Reply:
No, this is how it was done for Type 3. I've moved it down to the point where the XObjects are emitted.